time time
int32 odom1
int32 odom2
int32 odom3
int32 odom4
int32 mode
int32 tank_id
int32 track_point_id
int32 first_alignment
int32 laser_alignment
float32 distance_alignment
float32 Pillar_distance
int32 pause
int32 stop
int32 back_home
float32 other_car_x
float32 other_car_y
float32 other_car_theta
float32 infrared_right
float32 infrared_left
int32 is_start_camera
int32 next_target_num
int32 control1
int32 control2
int32 control3
int32 control4
